cfgov/v1/models/akamai_backend.py

```diff
@@ -68,22 +68,17 @@ def purge(self, url):
         resp.raise_for_status()
 
     def purge_all(self):
-        payload = self.get_payload(
-            obj=os.environ['AKAMAI_OBJECT_ID']
-        )
-        payload['type'] = 'cpcode'
-        payload['domain'] = 'production'
-        payload['action'] = 'invalidate'
-
+        obj = os.environ['AKAMAI_OBJECT_ID']
         resp = requests.post(
             os.environ['AKAMAI_PURGE_ALL_URL'],
             headers=self.headers,
-            data=json.dumps(payload),
+            data=json.dumps(self.get_payload(obj=obj)),
             auth=self.auth
         )
         logger.info(
-            u'Initiated site-wide Akamai flush, '
+            u'Attempted to invalidate content provider {obj}, '
```

Comment:
Is there a reason we're moving from a more human-readable message (that we're attempting to flush the entire site) to something a little more specific to language Akamai uses?  I understand this would be useful if we had more than one production site that Akamai was on, but it's only ever going to be for consumerfinance.gov -- right?

Reply:
Or if we want to include the ID for debugging purposes, can we still say - `Attempted a site-wide flush for site corresponding to CP code {obj}` ?